src/zcl_mustache_render.clas.abap

```diff
@@ -128,8 +128,9 @@ CLASS ZCL_MUSTACHE_RENDER IMPLEMENTATION.
                       it_data_stack = it_data_stack
                       iv_level      = lines( it_data_stack ) ). " Start from deepest level
 
+    DATA(t) = cl_abap_datadescr=>describe_by_data_ref( lr ).
     assign lr->* to <field>.
-    describe field <field> type lv_type.
+    lv_type = cl_abap_datadescr=>get_data_type_kind( <field> ).
```

Comment:
Hmmm, the describe is in fact faster if I remember correctly, that's why it was choosen in the first place.